filebeat/harvester/reader/multiline_test.go

```diff
@@ -73,6 +73,23 @@ func TestMultilineBeforeNegateOK(t *testing.T) {
 	)
 }
 
+func TestMultilineAfterNegateOKFlushPattern(t *testing.T) {
+	flushMatcher := match.MustCompile(`EventEnd`)
+
+	testMultilineOK(t,
+		MultilineConfig{
```

Comment:
could you add one more test where first there are lines which do not match the pattern `EventStart`?